### ISSUE TITLE
Defend the dwyl heart logo trademark

### ISSUE THREAD

**Issue by markwilliamfirth** [dwyl/hq#334]
#93 

Now that our trademark has been granted we need to defend it to retain it.

- [ ] Research similar logos

- [ ] Email the companies and challenge their use of our logo IP

Known similar logos:
- Fittle
- True View http://trueview.me/

@iteles are there any others that you know of already?

**Comment by iteles** [comment]
@markwilliamfirth I don't know of any others for now 👍 

Should we be doing this before we have a global trademark? If the orgs have business in the UK but are _based_ in another country, will this just prompt them to trademark the logo in their country?

Shall we open an issue on a global trademark?

**Comment by markwilliamfirth** [comment]
@iteles agreed each company should be thoroughly researched and the research should include their origins, base and where they trade before proceeding with any challenges. A global trademark might be necessary in future and we should open an issue.
